src/doc/src/reference/manifest.md

```diff
@@ -107,13 +107,18 @@ breaking change.
 
 <a id="the-authors-field-optional"></a>
 #### The `authors` field
-
-The optional `authors` field lists people or organizations that are considered
+The optional `authors` field lists in an array the people or organizations that are considered
 the "authors" of the package. The exact meaning is open to interpretation — it
 may list the original or primary authors, current maintainers, or owners of the
 package. An optional email address may be included within angled brackets at
 the end of each author entry.
 
+```toml
+[package]
+# ...
+authors = ["Graydon Hoare", "Fnu Lnu <no-reply@rust-lang.org>"]
```

Comment:
Not sure if it is a good idea to mention a specific person. Ferris might be better?
(not a big deal tho)

Reply:
Yeah, I didn't know what to do here for examples so I picked Graydon to perhaps amuse to folks who know or prompt those that don't to look into who they are given their importance with Rust history.  "Fnu Lnu" is a fake person.  Apparently, in some places this is common place holder name and means "First name unknown" and "Last name unknown" (Cf. https://en.wikipedia.org/wiki/Placeholder_name#Fnu_Lnu).